.env.example

```
@@ -1,25 +1,63 @@
+NEXT_PUBLIC_VERCEL_ENV = development
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)